packages/react/src/components/Footer/FooterNavGroup.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
```

Comment:
@photodow change these imports to point to the es folder.
cc: @kennylam @annawen1